components/Carbon.js

```diff
@@ -49,14 +46,16 @@ function getUnderline(underline) {
   return 'initial'
 }
 
-class Carbon extends React.PureComponent {
-  static defaultProps = {
+const Carbon = (
```

Comment:
Why does this component need to get converted to a functional component? If it isn't strictly required for upgrading to React@18, can we do these changes in a separate PR?